SoundSwitch/Framework/Configuration/SoundSwitchConfiguration.cs

```diff
@@ -66,7 +66,7 @@ public SoundSwitchConfiguration()
             BannerPosition = BannerPositionEnum.TopLeft;
             MaxNumberNotification = 5;
             BannerOnScreenTime = TimeSpan.FromSeconds(3);
-            PersistentMuteNotification = true;
+            MicrophoneMuteNotification = MicrophoneMuteEnum.None;
```

Comment:
```suggestion
            MicrophoneMuteNotification = MicrophoneMuteEnum.Persitent;
```

I want persistent to be the default value.
People can disable it later.

Reply:
resolved in latest commit